### DISCUSSION TITLE
how to run cron without IB key

### DISCUSSION THREAD

**Discussion by llchen223** [brndnmtthws/thetagang#545] [Q&A]
Running this as cron: 
```
In normal usage, you would run the script as a cronjob 
```
Is problematic, because this requires a manually ack through the IB key app on mobile phone.  How do I run cron without any manual intervention?

**Comment by brndnmtthws** [comment]
The trick involves creating a second IBKR login for the same account. You can lock down the permissions, and use it without IB key. There's a note here in the FAQ (under "IBKey and MFA-related authentication issues"): https://github.com/brndnmtthws/thetagang?tab=readme-ov-file#faq

I also made a note about it here: https://github.com/brndnmtthws/thetagang/issues/307#issuecomment-2123180521

And here's some documentation from another project (more detailed than what's currently in the README, I should link to it): https://github.com/Voyz/ibeam/wiki/Runtime-environment#using-a-secondary-account
